spec/features/users_creates_new_account_spec.rb

```diff
@@ -1,15 +1,15 @@
 require "rails_helper"
 
 feature "user creates new account" do
-  scenario "successfully" do
+  scenario "successfully without login" do
     visit new_user_path
 
     fill_in "user_name", with: "Mary"
     fill_in "user_email", with: "Mary@example.com"
     fill_in "user_password", with: "secret"
     click_on I18n.t("helpers.submit.user.create")
 
-    expect(page).to have_text(I18n.t("users.show.welcome", name: "Mary"))
```

Comment:
Does the page display "Mary"? If so, would be good to keep this assertion.

Reply:
No, it is generic